graf3d/eve7/inc/ROOT/REveVSDStructs.hxx

```diff
@@ -17,7 +17,7 @@
 #include <ROOT/REveVector.hxx>
 
 ////////////////////////////////////////////////////////////////////////////////
-/// VSD Structures
+// VSD Structures
```

Comment:
```suggestion
/// \file
/// \brief Basic structures for Reve VSD concept
```

Reply:
I would prefer not to change anything more than necessary. Also the documentation overview was never associated to the file AFAICT...